### PR TITLE
[#1336] Show non-blank line count in the brackets instead of blank lines

### LINKED ISSUE #1336: Show non-blank lines count instead of blank-lines count
Current: blank lines count is shown within brackets
image

Suggestion: It's more useful to show the non-blank links count instead? 

Reason: To save the user from having to do the mental calculation.

### PR DESCRIPTION
**Pull request by lm5lm5**
Fixes #1336
Please note that the change to code done by this PR is the same as PR #1339. 
This PR is made to resolve the unrecognised committer issue in PR #1339.
```
Currently, the report in the code panel (authorship) part shows
the number of blank lines written by an author in a file or in all
files of a file type inside the brackets beside the total number of
lines written. The viewer needs to do mental calculation to get the
number of non-blank lines written by the author.

Let's change the front end code to show non-blank line count instead,
and update the user guide accordingly.
```
Before:
before
After:
after
The new image to be used in the user guide:
code-panel-new

### CODE REVIEW

**Comment by Tejas2805** [comment]
You can close the other PR.

**Review by Tejas2805 — APPROVED** [review]

**Review by jamessspanggg — APPROVED** [review]

**Comment by lm5lm5** [comment]
> You can close the other PR and change the title of this PR accordingly.

I have closed the other PR and thank you for changing the title of this PR.

**Review by fzdy1914 — CHANGES_REQUESTED** [review]
Since you are already touching this part. Can you help part this part in UG also?

image

**Comment by lm5lm5** [comment]
> Since you are already touching this part. Can you help part this part in UG also?
> 
> image

Sure, I'll work on that this week.

**Comment by lm5lm5** [comment]
@fzdy1914 I have updated the corresponding UG section. Please check. Thank you.

File: docs/ug/usingReports.md [hunk 1]
```diff
@@ -126,13 +126,15 @@ Notes:<br>
...
-<img src="../images/code-panel.png" alt="code panel" width="468">
+<img src="../images/code-panel-new.png" alt="code panel" width="468">
 
 The `Code panel` allows users to see the code attributed to a specific author. Click on the `</>` icon beside the name of the author in the `Chart panel` to display the `Code panel` on the right.
 * The Code panel shows the files that contain author's contributions, sorted by the number of lines written.
 * Select the radio button to enable one of the following 2 filters. Note that only 1 of the 2 filters is active at any time.
     * Type file path glob in glob filter to include files matching the glob expression.
-    * Select the checkboxes to include files of preferred file extensions.
+    * Select the checkboxes to include files of preferred file extensions. 
+    The number right beside the file extension shows the total number of lines written by the author in the files of such file extension,
```
Comment (Tejas2805) [inline docs/ug/usingReports.md:136]:
```suggestion
     The number right beside the file extension shows the total number of lines written by the author in such file extension files. In contrast, the number inside the parenthesis indicates the number of non-blank lines written by the author.

```

Can update this accordingly.

File: docs/ug/usingReports.md [hunk 2]
```diff
@@ -126,13 +126,15 @@ Notes:<br>
...
+    while the number inside the parenthesis shows the number of non-blank lines written by the author.
```
Comment (Tejas2805) [inline docs/ug/usingReports.md:137]: Can get rid of this then.

File: docs/ug/usingReports.md [hunk 3]
```diff
@@ -126,13 +126,15 @@ Notes:<br>
 <!-- ==================================================================================================== -->
 
 ## Code panel
-<img src="../images/code-panel.png" alt="code panel" width="468">
+<img src="../images/code-panel-new.png" alt="code panel" width="468">
```
Comment (Tejas2805) [inline docs/ug/usingReports.md:129]: Any specific reason for not replacing the original image?

**Review by Tejas2805 — COMMENTED** [review]
LGTM! Just some nits.

File: docs/ug/usingReports.md (hunk 3 above)
Comment (lm5lm5) [reply docs/ug/usingReports.md:129]: I would like to check if you are good with the new image.
I will remove the original image if you agree on the new one.

**Comment by lm5lm5** [comment]
I make some commits just now and it seems that Travis CI build keeps running and never completes. What could be the issue?
image

**Comment by lm5lm5** [comment]
@Tejas2805 I have made the changs and replace the image.

**Comment by Tejas2805** [comment]
LGTM! @fzdy1914 Your review is needed.

**Review by fzdy1914 — APPROVED** [review]

**Pull request merged**